TbsCore/Helpers/SitterHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace TravBotSharp.Files.Helpers
+{
+    public static class SitterHelper
+    {
+        public static bool isSitter(HtmlAgilityPack.HtmlDocument htmlDoc, Classificator.ServerVersionEnum version)
```

Comment:
do we really need SitterHelper just for this 1 method? Just move this method into AccountHelper. And use **IsSitter** not **isSitter**